src/views/well-architected-framework/index.tsx

```diff
@@ -0,0 +1,49 @@
+import slugify from 'slugify'
+import SidebarSidecarLayout from 'layouts/sidebar-sidecar'
+import { generateTopLevelSidebarNavData } from 'components/sidebar/helpers'
+import ProductViewContent from 'views/product-tutorials-view/components/product-view-content'
+import HeroHeadingVisual from 'views/product-landing/components/hero-heading-visual'
+import { SidebarProps } from 'components/sidebar'
+import OverviewCta from 'views/product-landing/components/overview-cta'
+import { generateWafSidebarData } from './utils/generate-sidebar-items'
+import wafContent from 'content/well-architected-framework/index.json'
+import s from './well-architected-framework-landing.module.css'
+
+export const wafData = {
```

Comment:
I wonder if this should live in a different place 🤔

Reply:
💭 I think a new file in `src/data/` would be reasonable. It's where we keep non-authorable things at the moment and the folder name isn't specific to only allowing product data.

Reply:
(No action needed in this PR if you still want to think about it!)

Reply:
I think that is the right call, I'll do that now.